docker/README.md

```diff
@@ -69,3 +67,10 @@ docker run --cap-add MKNOD --cap-add SYS_ADMIN --device /dev/fuse \
 docker volume rm redox-"$(id -u)-$(id -g)"-cargo \
```

Comment:
I think this section should be extended to the rustup volume.

Reply:
Not sure I get what you mean. Can you elaborate?

Reply:
People might also want to remove the volume `redox-"$(id -u)-$(id -g)"-rustup`.

I guess the `Clear the named volume containing the cargo cache` section could be generalized to `Clear the named volumes`. And a command added to clean the volume `redox-"$(id -u)-$(id -g)"-rustup`.

Reply:
Ah, it was already there in the next line, the preview window here cut it out. The headline was referring to cargo only though.
Fixed it.